project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,685 @@
+            - not: Windows|Triple, Low-E, Insulated, Argon, L-Gain
+        costs:
+          - value: 46.30 
+            multiplier: Window Area (ft^2)
+        lifetime: 30
+      - &infiltration_30pct_reduction_windows
+        option: Infiltration Reduction|30%
+        apply_logic:
+          - and:
+            - not: Vintage|2000s
+            - not: Vintage|2010s
+            - or:
+              - Windows|Single, Clear, Metal, Exterior Low-E Storm
+              - Windows|Single, Clear, Non-metal, Exterior Low-E Storm
+              - Windows|Single, Clear, Metal
+              - Windows|Single, Clear, Metal, Exterior Clear Storm
+              - Windows|Single, Clear, Non-metal
+              - Windows|Single, Clear, Non-metal, Exterior Clear Storm
+        costs:
+          - value: 0.53 # from LBNL data for general air sealing (https://doi.org/10.20357/B7FP4D).
+                        # Interpolated between 20% and 40% reductions to get $0.44/ft2,
+                        # then used 1.2052 multiplier to inflate from Jan 2019 to April 2023 https://www.bls.gov/data/inflation_calculator.htm
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &infiltration_15pct_reduction_windows_thin_triple
+        option: Infiltration Reduction|15%
+        apply_logic:
+          - and:
+            - not: Vintage|2000s
+            - not: Vintage|2010s
+            - or:
+              - Windows|Double, Clear, Thermal-Break, Air
+              - Windows|Double, Low-E, H-Gain
+              - Windows|Double, Low-E, L-Gain
+              - Windows|Double, Clear, Metal, Exterior Low-E Storm
+              - Windows|Double, Clear, Non-metal, Exterior Low-E Storm
+              - Windows|Double, Clear, Metal, Air
+              - Windows|Double, Clear, Metal, Air, Exterior Clear Storm
+              - Windows|Double, Clear, Non-metal, Air
+              - Windows|Double, Clear, Non-metal, Air, Exterior Clear Storm
+              - Windows|Double, Low-E, Non-metal, Air, M-Gain
+              - Windows|Double, Low-E, Non-metal, Air, L-Gain
+        costs: &costs_infiltration_15pct_reduction
+          - value: 0.35 # from LBNL data for general air sealing (https://doi.org/10.20357/B7FP4D).
+                        # liinear Interpolated for the 20% (0.34) and 40% reductions (0.54) to get $0.29/ft2,
+                        # then used 1.2052 multiplier to inflate from Jan 2019 to April 2023 https://www.bls.gov/data/inflation_calculator.htm
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf      
+
+  - upgrade_name: EnergyStar Windows
```

Comment:
There is something wrong in this upgrade but I haven't been able to figure out what. The upgrades analyzer is showing 99.8% of models are getting an upgrade, but also shows that 1.6% of models in the southern climate zone are triple pane (the Non-metal, air, L-gain option) and aren't being upgraded, so only 98.4% of the models should be being upgraded. But I can't find the error in the yaml.

Reply:
Haha nevermind, I figured it out. Looks like there are almost no triple paned windows in the southern climate zone in baseline, which makes sense.